design/design.md

```diff
@@ -0,0 +1,251 @@
+# ALBA implementation design document
+
+## Introduction
+
+This is the first design doc for our implementation of Approximate Lower Bound
+Arguments intended to guide the development of the library.
+See the [website](https://alba.cardano-scaling.org) or
+the [paper](https://eprint.iacr.org/2023/1655) to get an overview of this
+cryptographic primitive.
+
+We intend to create a high quality reference implementation of ALBA sufficient
+for accurately prototyping larger protocols utilizing this primitive.
+Applications that could benefit, at least in the Cardano universe, include
+Mithril, Peras, Leios and cross-chain bridges.
```

Comment:
repplace `cross-chain bridges` with `partner chains` or `sidechains`

Reply:
"... Leios and bridges with partner chains" ?